### DISCUSSION TITLE
Artifact dice

### DISCUSSION THREAD

**Discussion by guldfink** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#23] [Q&A] [answered]
How do I add an artifacts dice to a roll, for instance connected to a talent or artifact?

**Comment by aMediocreDad** [comment, marked as answer]
Hey there @guldfink!

You can either use the roll modifier on a talent or artifact. 
![2021-02-07 (1)](https://user-images.githubusercontent.com/9851733/107151361-7f657880-6962-11eb-88ae-5a92d4bf4bed.png)
Or, you can add it manually during the roll by inputting "d8", "d10", or "d12", in the "Artifacts" input box during a roll.
![2021-02-07](https://user-images.githubusercontent.com/9851733/107151375-8d1afe00-6962-11eb-845f-786924ddb6dd.png)

**Reply by guldfink** [reply]
Thank you for your answer! I created a talent with a roll modifier, but when I roll I can only see the d6s.

**Reply by guldfink** [reply]
No, sorry! Now it worked! I used capitals before!

**Reply by aMediocreDad** [reply]
> No, sorry! Now it worked! I used capitals before!

Yeah, this is something we could probably fix. The system should allow you to use both lower and upper case.
